tests/instances.spec.ts

```diff
@@ -238,3 +239,23 @@ test("Duplicate an instance", async ({ page }) => {
   await page.waitForSelector(`text=Created instance ${instance}-duplicate.`);
   await deleteInstance(page, `${instance}-duplicate`);
 });
+
+test("Create an image from an instance", async ({ page }) => {
+  await page.goto("/ui/");
+  await visitInstance(page, instance);
+  const instanceBaseImageName = await page
+    .locator("text=Base image")
+    .locator("xpath=following-sibling::*")
+    .innerText();
+  await page.getByRole("button", { name: "Create image" }).click();
+  await page
+    .getByRole("dialog", { name: "Create image from instance" })
+    .getByRole("button", { name: "Create image" })
+    .click();
+
+  await page.waitForSelector(`text=Image created from instance ${instance}.`);
+
+  //Images created from an instance will have the same Image name and therefore deleteImage should be called twice.
+  await deleteImage(page, instanceBaseImageName);
+  await deleteImage(page, instanceBaseImageName);
```

Comment:
I think we shouldn't delete the base image here, only the image we created from the instance. We should give that image an alias on creation, so we can identify and delete only that one.

Reply:
To clarify, this section deletes the image created upon instance creation, as well as the image created by clicking through Create Image. The instanceBaseImageName gets the name of the first image (which I have called the base image) from the instances page.

EDIT: Giving the image an alias on creation is a great idea and it has remidned me that I have actually already created this function through createImageFromInstance. As such, I have refactored this test to include this function, as well as get an image alias back and delete both images via both alias and image name.

EDIT 2: So we shouldn't delete the base image, here? No cleanup?

Reply:
EDIT 3: Perhaps we shouldn't delete the base image because due to the structure of the test suite, both an instance and vm instance are created before each test and for thorough cleanup, one would have to delete the image from both. Which is not impossible, but is out of scope.
I would be happy to add this cleanup functionality if you see it fit though, it would only be a few more lines of code in the .afterAll test.

Reply:
I think we should keep the base images and never remove them. This will speed up the tests. Cleaning them up means the next test needs to download the base image again. I think it is fine to keep the base image around.

If we create custom images from instances or by upload with alias, we should clean them up.